packages/evolution-common/src/services/widgets/WidgetConfig.ts

```diff
@@ -114,10 +114,14 @@ export type InputCheckboxType<CustomSurvey, CustomHousehold, CustomHome, CustomP
     seed?: number;
     shuffle?: boolean;
     addCustom?: boolean;
-    columns?: number;
     sameLine?: boolean;
     customLabel?: I18nData<CustomSurvey, CustomHousehold, CustomHome, CustomPerson>;
     datatype?: 'string' | 'integer' | 'float' | 'text';
+    alignChoices?: boolean;
```

Comment:
I did not notice the first time I think, but alignChoices is redundant with the presence of either rows and columns. We should not require to set alignChoices if we have columns or rows, just as alignChoices = true does not make sense if the other values are not set. So we should have columns, rows, customAlignmentLength. And the alignment field to specify which way to align. Also, there should be a default that if there is no columns, rows or customalignmentLength, but alignment is set, then vertical means 1 column, horiziontal means 1 row and auto is... 1 column?